app/models/chat_message.rb

```diff
@@ -68,44 +98,6 @@ def rebake!(invalidate_oneboxes: false, priority: nil)
 
     Jobs.enqueue(:process_chat_message, args)
   end
-
-  COOK_FEATURES = {
-    anchor: true,
-    "auto-link": true,
-    bbcode: true,
-    "bbcode-block": true,
-    "bbcode-inline": true,
-    "bold-italics": true,
-    "category-hashtag": true,
-    censored: true,
-    checklist: false,
-    code: true,
-    "custom-typographer-replacements": false,
-    "d-wrap": false,
-    details: false,
-    "discourse-local-dates": true,
-    emoji: true,
-    emojiShortcuts: true,
-    html: false,
```

Comment:
Not sure why this was added but this had no effect on anything.

https://github.com/discourse/discourse/blob/04c77766506184b3b0daccddc51699d34e4eef9e/app/assets/javascripts/pretty-text/addon/engines/discourse-markdown-it.js#L376